data/avgi/avgi 1 first contact.txt

```diff
@@ -0,0 +1,1055 @@
+				clear "avgi: asked about aberrant"
+
+mission "Avgi: Humans From the Deep"
+	name "Humans Among the Avgi"
+	description "Visit the <destination>, where there is another human, Darius Calwell, stranded in Avgi space."
+	landing
+	source "Peripheria"
+	destination "Aktina Cylinder"
+	to offer
+		has "Avgi: First Contact: done"
+
+
+
+mission "Avgi: Defend Ensemble 1"
+	name "Avgi Strike Force"
+	description "Meet up with the Avgi strike force at <destination>."
+	landing
+	passengers 1
+	blocked "You'll need a free bunk on the <ship> if you want to take Darius anywhere off this station."
+	source "Aktina Cylinder"
+	destination "Navigeo Yards"
+	to offer
+		has "Avgi: Humans From the Deep: done"
+	on offer
+		log "People" "Darius Caldwell" `Darius was a Deep Security captain, who led an expedition sponsored by the Deep to learn more about Quarg technology. While on the return trip, he ordered his crew to investigate an anomalous extra hyperlane leading south of human space. After becoming trapped in Avgi space, the Deep crew sought refuge with the Consonance. Darius seems to blame himself for his crew's situation.`
+		conversation
+			`From afar, the Aktina Cylinder looks like a titanic skeletal tube, with ribs of graphene and a metallic spine. Following the instructions of the station's control authority, you fly through the station's open frame to its small pressurized area, which houses the construction workers and maintenance crews.`
+			`	You've been told that word of your arrival has been sent ahead of you, so you aren't entirely surprised to see a middle-aged human waiting for you, a look of pure astonishment on his face as your ship settles into a free hangar. You disembark from the <ship> to find Darius waiting right outside the hatch.`
+			choice
+				`	"Hello there."`
+				`	"Greetings, human. I come in peace."`
+			`	The man blinks in surprise, then extends a hand, which you shake. "So, another one of us here in Fairyland. I don't know if I should be glad to see another friendly face, or feel bad that another soul is stranded here along with us." He shakes his head. "Let's find somewhere to talk. If you don't mind telling me, how'd you end up here anyway?"`
+			choice
+				`	"I have a jump drive."`
+				`	"I have a device that can bypass hyperlanes."`
+			`	He somehow manages to look even more astonished as you walk. "A jump drive? Where the hell did you get one of those?"`
+			choice
+				`	"I took it from some hostile aliens near the Core."`
+					goto korath
+					to display
+						"reputation: Korath" < -10
+				`	"I took it from some aliens in the Deneb system."`
+					goto pug
+					to display
+						has "FWC Pug 2D: done"
+				`	"I got it from the Syndicate, who took it from an alien race near the Core."`
+					goto korath
+					to display
+						has "FW Pug 2B: done"
+				`	"I took it from some angry space squirrels."`
+					goto unfettered
+					to display
+						has "Wanderers: Translation Machine: done"
```

Comment:
You can get a jump drive from the Unfettered without having done this mission, or even having visited Wanderer space,